### DISCUSSION TITLE
Separate taskbars

### DISCUSSION THREAD

**Discussion by Blueeyy** [valinet/ExplorerPatcher#301] [Q&A] [answered]
Hi, is there a way to have separate taskbars on multiple monitor setups?
Right now all my monitors use the same taskbar and it gets pretty full when i try to use multiple apps on multiple monitors

**Comment by valinet** [comment, marked as answer]
All these settings work:

![image](https://user-images.githubusercontent.com/6503598/140819159-6f341f46-cd65-4482-a2fd-bd573ae23df3.png)

Thanks

**Comment by Blueeyy** [comment]
Oh i thought i was supposed to use the properties from win + x
didn't think windows settings would work

thank you!

**Reply by valinet** [reply]
Yeah, no worries, you're welcome. The Properties GUI mainly does not duplicate functionality offered through there, as that is still easily accessed by right clicking the taskbar and choosing "Taskbar settings". And like, for some I did not bother bringing them over, whatever... once you know they're there, it's not that big of a deal, I think...

Thanks
